### PR TITLE
virttest/utils_test/qemu.py: Add check Link layer for rdma migration

### PR DESCRIPTION
**Pull request by spcui**
Add check Link layer step for rdma migration.

Signed-off-by: Shuping Cui <scui@redhat.com>

ID: 1253528

### CODE REVIEW

File: virttest/utils_test/qemu.py [hunk 1]
```diff
@@ -1245,6 +1245,21 @@ class MultihostMigrationRdma(MultihostMigration):
     def __init__(self, test, params, env, preprocess_env=True):
         super(MultihostMigrationRdma, self).__init__(test, params, env,
                                                      preprocess_env)
+        check_link_layer_cmd = "ibstat"
```
Comment (xutian) [inline virttest/utils_test/qemu.py]: avocado.utils.find_command("ibstat") ?

File: virttest/utils_test/qemu.py [hunk 2]
```diff
@@ -82,6 +84,31 @@ def get_numa_status(numa_node_info, qemu_pid, debug=True):
...
 
 
+def get_link_layer(params, cmd):
+    """
+    Get host link layer
+
+    :param params: Dictionary with the test parameters.
+    :param cmd: Command string
+    """
+    try:
```
Comment (xutian) [inline virttest/utils_test/qemu.py]: No need try this exception and reraise it again.

File: virttest/utils_test/qemu.py [hunk 3]
```diff
@@ -82,6 +84,31 @@ def get_numa_status(numa_node_info, qemu_pid, debug=True):
...
+        utils_path.find_command(cmd)
+    except utils_path.CmdNotFoundError:
+        raise exceptions.TestNAError("Can't find command '%s'" % cmd)
+
+    expected_link_layer = params.get("expected_link_layer",
```
Comment (xutian) [inline virttest/utils_test/qemu.py]: InfiniBand is a  network adapter vendor, so it looks rename link_layer to nic_vendor looks more  reasonable

**Comment by spcui** [comment]
@xutian updated.

File: virttest/utils_test/qemu.py [hunk 4]
```diff
@@ -82,6 +84,28 @@ def get_numa_status(numa_node_info, qemu_pid, debug=True):
     return (qemu_memory, qemu_cpu)
 
 
+def get_nic_vendor(params, cmd):
+    """
+    Get host link layer
+
+    :param params: Dictionary with the test parameters.
+    :param cmd: Command string
+    """
```
Comment (suqinhuang) [inline virttest/utils_test/qemu.py]: is this step necessary?
"# echo 0000:07:00.0 ib ib > /etc/rdma/mlx4.conf"

Comment (spcui) [inline virttest/utils_test/qemu.py]: no

**Comment by suqinhuang** [comment]
LGTM
Acked-by: Suqin Huang

**Comment by CongSmile** [comment]
Looks good.

Acked-by: Cong Li <coli@redhat.com>

**Comment by spcui** [comment]
@xutian could you help review this one, thx?

**Pull request merged**